pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -696,6 +693,7 @@ protected void startConnection(Function<String, Boolean> connectionDone) {
   private boolean attemptConnection() {
     try {
       isConnected = false;
+      deviceManager.pause();
```

Comment:
Similar to the other comment -- here's pause() -- where's the matching resume()?